clients/src/test/java/org/apache/kafka/common/security/oauthbearer/secured/AccessTokenBuilder.java

```diff
@@ -49,22 +49,24 @@
 
     private Long expirationSeconds;
 
-    private RsaJsonWebKey jwk;
+    private PublicJsonWebKey jwk;
 
-    public AccessTokenBuilder() throws JoseException {
+    public AccessTokenBuilder() {
         this(new MockTime());
     }
 
-    public AccessTokenBuilder(Time time) throws JoseException {
+    public AccessTokenBuilder(Time time) {
         this.issuedAtSeconds = time.milliseconds() / 1000;
         this.expirationSeconds = this.issuedAtSeconds + 60;
-        this.jwk = createJwk();
     }
 
-    public static RsaJsonWebKey createJwk() throws JoseException {
-        RsaJsonWebKey jwk = RsaJwkGenerator.generateJwk(2048);
-        jwk.setKeyId("key-1");
-        return jwk;
+    public String alg() {
+        return alg;
+    }
+
+    public AccessTokenBuilder alg(String alg) {
```

Comment:
While you are at this, a few other methods like audience() and subjectClaimName seem never used. Is that expected?

Reply:
They're there for completeness. I've removed them now.